openbas-api/src/main/java/io/openbas/rest/team/form/TeamUpdateInput.java

```diff
@@ -3,54 +3,29 @@
 import static io.openbas.config.AppConfig.MANDATORY_MESSAGE;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.validation.constraints.NotBlank;
 import java.util.ArrayList;
 import java.util.List;
+import lombok.Data;
 
+@Data
 public class TeamUpdateInput {
 
   @NotBlank(message = MANDATORY_MESSAGE)
   @JsonProperty("team_name")
+  @Schema(description = "Name of the team")
```

Comment:
We should probably create a super class here to mutualize code between the various team input types

Reply:
Good point !